WeakAuras/Init.lua

```diff
@@ -56,6 +56,14 @@ function WeakAuras.IsRetail()
   return WOW_PROJECT_ID == WOW_PROJECT_MAINLINE
 end
 
+function WeakAuras.IsBCCAndNotWrath()
+  return WeakAuras.IsBCC() and not WeakAuras.IsWrathClassic()
```

Comment:
I would expect IsBCC to already not include Wrath, why doesn't that work?

Reply:
left over code i'll delete this function (not use anywhere)

Reply:
ha no it's changed next commit

Reply:
WOW_PROJECT_ID == WOW_PROJECT_BURNING_CRUSADE_CLASSIC on Wrath

Adding this function on vehicle commit made me realize the next commit that change all version checks had to be done